src/Bellatrix.Web/services/DevToolsService.cs

```diff
@@ -28,6 +30,99 @@ public DevToolsService(IWebDriver wrappedDriver)
 
     public DevToolsSessionDomains DevToolsSessionDomains { get; set; }
     public DevToolsSession DevToolsSession { get; set; }
+    public List<NetworkRequestSentEventArgs> RequestsHistory { get; set; }
+    public List<NetworkResponseReceivedEventArgs> ResponsesHistory { get; set; }
+
+    public void StartMonitoringBiDiApi()
+    {
+        RequestsHistory = new();
+        ResponsesHistory = new();
+
+        INetwork networkInterceptor = WrappedDriver.Manage().Network;
+        networkInterceptor.NetworkResponseReceived += (o, s) =>
+        {
+            ResponsesHistory.Add(s);
+        };
+        networkInterceptor.NetworkRequestSent += (o, s) =>
+        {
+            RequestsHistory.Add(s);
+        };
+
+        networkInterceptor.StartMonitoring();
+    }
+
+    public void EmptyResponseAndRequestHistory()
```

Comment:
check the other Proxy service what is the name for this, but nowhere we use just empty but for sure Clear also I guess the words should be in plural
ClearRequestsReponsesHistory

Reply:
Method renamed to ClearRequestsResponsesHistory